### ISSUE TITLE
[Android Renderer] TextBlock with two links does not property receive focus

### ISSUE THREAD

**Issue by anna-dingler** [microsoft/AdaptiveCards#8077]
# Platform

* Android

# Issue

Example TextBlock: "this is a string [link](https://github.com/microsoft/AdaptiveCards/pull/url) and this is another [link](https://github.com/microsoft/AdaptiveCards/pull/url)"

This scenario is not properly handled in the Android renderer. If a TextBlock contains a url, we allow the entire control to receive focus. If there are two separate links, we cannot navigate between them, and pressing "Enter" does not activate either link.

Seems like the problem here is that URLSpan is not a view, so we cannot use the SetFocus method.
